extensions/stats/config.h

```diff
@@ -0,0 +1,384 @@
+// response_code="200",
+// response_flags="-",
+// source_app="svc01-0",
+// source_principal="unknown",
+// source_version="v2",
+// source_workload="svc01-0v2",
+// source_workload_namespace="service-graph01"
+// }
+
+std::vector<Mapping> getStandardMappings() {
+  return {
+      MAPPING("reporter", (outbound ? vSource : vDest)),
+      // --> Peer info source
+      MAPPING("source_workload", source.workload_name()),
+      MAPPING("source_workload_namespace", source.namespace_()),
+      MAPPING("source_principal", requestInfo.source_principal),
+      MAPPING("source_app", source_labels["app"]),
+      MAPPING("source_version", source_labels["version"]),
+
+      // --> Peer info destination
+      MAPPING("destination_workload", dest.workload_name()),
+      MAPPING("destination_workload_namespace", dest.namespace_()),
+      MAPPING("destination_principal", requestInfo.destination_principal),
+      MAPPING("destination_app", dest_labels["app"]),
+      MAPPING("destination_version", dest_labels["version"]),
+
+      // --> Service info
+      MAPPING("destination_service_host", requestInfo.destination_service_host),
+      MAPPING("destination_service_name", dest.workload_name()),
+      MAPPING("destination_service_namespace", dest.namespace_()),
+
+      MAPPING("request_protocol", requestInfo.request_protocol),
+      MAPPING("response_code", requestInfo.response_code),
+      MAPPING_SYM("response_flags", requestInfo.response_flag, vDash),
+
+      MAPPING("connection_security_policy",
+              (outbound ? unknown : (requestInfo.mTLS ? vMTLS : vNone))),
+      MAPPING_SYM("permissive_response_code",
+                  requestInfo.permissive_response_code, vNone),
+      MAPPING_SYM("permissive_response_policyid",
+                  requestInfo.permissive_response_policyid, vNone)};
+}
+
+// StatGen is dimensioned using standard Istio dimensions.
+// Standard Istio metrics have the following dimensions
+class StatGen {
+ public:
+  StatGen(std::string name, MetricType metricType, valueExtractorFn valueFn)
+      : name_(name), valueFn_(valueFn) {
+    metric_ =
+        std::make_shared<Metric>(metricType, name, mappings_.metricTags());
+  };
+
+  StatGen() = delete;
+  inline StringView name() const { return name_; };
+
+  // return a SimpleSharedPtr
+  SimpleStatSharedPtr resolve(bool outbound, const common::NodeInfo& source,
+                              const common::NodeInfo& dest,
+                              const Common::RequestInfo& requestInfo) {
+    auto vals = mappings_.eval(outbound, source, dest, requestInfo);
+    auto metric_id = metric_->resolveWithFields(vals);
+    logDebug(absl::StrCat(__FUNCTION__, ":", __LINE__, ":", metric_id, "/",
+                          source.name(), dest.name(), requestInfo.mTLS));
+    return std::make_shared<SimpleStat>(metric_id, valueFn_);
+  };
+
+ private:
+  std::string name_;
+  valueExtractorFn valueFn_;
+  MetricSharedPtr metric_;
+  Mappings mappings_;
```

Comment:
mappings_ is private and I didn't see it assigned... Did I lose anything?

Reply:
mappings_ default constructor creates istioStandard metrics mappings.